### ISSUE TITLE
create syntax to compare dseries to numbers

### ISSUE THREAD

**Issue by houtanb** [DynareTeam/dseries#4]
e.g. if ```a``` is a dseries, the command ```a>0``` would return a vector/matrix of all elements in a.data that are greater than 0

**Comment by stepan-a** [comment]
I do not think that this would be a good move. Usually this binary op returns booleans (true/false or 1/0) in matlab. I think that we should be consistent with this behavior. You can get what you want with:
```matlab
a(a.dates(find(a.data>0)))
```
We could implement something shorter like:
```matlab
a(a>0)
```
We would need to overload >, <, >=, <= and to modifty the overloaded subsref method (so that we can index ```dseries```objects with logical arrays). But all this would only make sense for ```dseries``` objects having only one variable, so I do not know if it's a good idea...

**Comment by houtanb** [comment]
Yes, I misspoke in the initial message. I wanted the same functionality as in Matlab. To return a vector/matrix with all elements **marked** as 1 that are greater than 0. This is useful for a dseries with more than one variable

**Comment by stepan-a** [comment]
Yes, we can implement the comparison operators (```a>0```) for ```dseries``` objects with more than one variable, but I don't know what should return ```a(a>0)``` in this case (the returned ```true``` and ```false``` won't be, a priori, on the same rows for each variable).

That said, this will allow us to filter a ```dseries``` object with respect to the sign of one of the variables:
```matlab
a(a.x>0)
```  
or with respect to any rule applied on a variable
```matlab
a(fun(a.x))
```
where ```fun```is a function returning a vector of booleans.

**Comment by houtanb** [comment]
Once this is done, fix code in DynareTeam/reporting@4083eeee521e8f7367073c50955599a3ee72bdc7
